src/strings/PowerFxResources.en-US.resx

```diff
@@ -4076,6 +4076,10 @@
   <data name="ErrUDF_FunctionAlreadyDefined" xml:space="preserve">
     <value>Function {0} is already defined.</value>
     <comment>This error message shows up when user tries to define a function that already exists</comment>
+  </data>
+  <data name="WrnUDF_ShadowingBuiltInFunction" xml:space="preserve">
+    <value>Warning: User defined function {0} will shadow the existing built-in function {0}.</value>
```

Comment:
>User defined [](http://example.com/codeflow?start=20&length=12)

I'm almost sure that we should have a hyphen here (`User-defined function`, instead of `User defined function`) - see [https://en.wikipedia.org/wiki/User-defined_function](https://en.wikipedia.org/wiki/User-defined_function), for example. Something to check with the content team and/or PM. Same with other resources that use this term.

Reply:
Copilot also agrees with this :smile::

Both “user-defined function” and “user defined function” are grammatically correct, but they convey slightly different meanings. Let me explain:

1. “**User-Defined Function**”: This form emphasizes that the function is specifically created or defined by a user (or programmer). It suggests that the function is not part of the built-in language features or libraries.
2. “**User Defined Function**”: In this form, “user” serves as an adjective describing the type of function. It implies that the function is associated with or related to users, without necessarily emphasizing the act of defining it.